exchanges/sharedtestvalues/sharedtestvalues.go

```diff
@@ -103,3 +109,42 @@ func SkipTestIfCannotManipulateOrders(t *testing.T, exch exchange.IBotExchange,
 func AreAPICredentialsSet(exch exchange.IBotExchange) bool {
 	return exch.VerifyAPICredentials(exch.GetDefaultCredentials()) == nil
 }
+
+// EmptyStringPotentialPattern is a regular expression pattern for a potential
+// empty string into float64
+var EmptyStringPotentialPattern = `.*float64.*json:"[^"]*,string".*`
+
+// ForceFileStandard will check all files in the current directory for a regular
+// expression pattern. If the pattern is found the test will fail.
+func ForceFileStandard(t *testing.T, pattern string) error {
+	t.Helper()
+
+	r := regexp.MustCompile(pattern)
+
+	root := "." // Specify the root directory to start walking from
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !info.IsDir() && strings.HasSuffix(path, ".go") {
+			fileContents, err := ioutil.ReadFile(path)
```

Comment:
```suggestion
			fileContents, err := os.ReadFile(path)
```

As the package ioutil is deprecated https://pkg.go.dev/io/ioutil